lib/validic/response.rb

```diff
@@ -1,9 +1,11 @@
 module Validic
   class Response
+    include REST::Request
     attr_reader :summary, :records, :attributes
-    def initialize(summary, response)
-      @summary = summary
 
+    def initialize(summary, response, client = nil)
+      @summary = summary
+      @client = client if client
```

Comment:
the connection gives us an ability to make the call for the next/previous set of records